File: leetcode/python/2114.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max_num = 0

        for s in sentences:
            cnt = 0
            for w in s:
                if w == ' ':
                    cnt += 1
            max_num = max(cnt, max_num)

        return max_num + 1

File: leetcode/java/2114.java
class Solution {
  public int mostWordsFound(String[] sentences) {
    int sum = 0;
    for (String s : sentences) {
      int count = 0;
      for (char w : s.toCharArray()) {

        if (w == ' ') {
          count++;
          sum = Math.max(count, sum);
        }
      }
    }
    return sum + 1;
  }
}
